docs/en/siem/siem-ui.asciidoc

```diff
@@ -86,15 +87,14 @@ Interactive widgets let you drill down for deeper insights:
 
 There are also tabs for viewing and investigating specific types of data:
 
-* Flows
-** Source IPs
-** Destination IPs
-** Source Countries
-** Destination Countries
-* DNS
-* TLS
-* Anomalies
-* <<det-engine-terminology, External alerts>>
+* Flows: source and destination IP addresses and countries
+* DNS: DNS network queries
+* HTTP: received HTTP requests (HTTP requests for applications using
+{apm-get-started-ref}/overview.html[APM] are monitored by default)
```

Comment:
If we're talking about the product here, and not the use case (which I think is what we're doing) this should be **Elastic APM** instead.